ui/src/features/Dialog/components/settings/AccountDialog.tsx

```diff
@@ -58,17 +77,42 @@ const AccountDialogContent: React.FC = () => {
                   onChange={e => setEmail(e.target.value)}
                 />
               </div>
+              <div>
+                <Label htmlFor="password">{t("Password")}</Label>
+                <Input
+                  id="password"
+                  placeholder={t("Password")}
+                  disabled={isLoading}
+                  value={password}
+                  type="password"
+                  onChange={e => setPassword(e.target.value)}
+                />
+              </div>
+              <div>
+                <Label htmlFor="confirm-password">{t("Confirm Password")}</Label>
+                <Input
+                  id="confirm-password"
+                  placeholder={t("Confirm Password")}
+                  disabled={isLoading}
+                  value={passwordConfirmation}
+                  type="password"
+                  onChange={e => setPasswordConfirmation(e.target.value)}
+                />
+              </div>
               <div>
                 <Button
                   className="self-end"
-                  disabled={isLoading || loading || !userName || !email}
+                  disabled={isLoading || loading || !name || !email}
```

Comment:
This means the user must update name or email at the same time as password to update password. Please add ` || (password && password !==  passwordConfirmation)`

Reply:
Oh no no, it means they can't make them empty. If they are left as it is, nothing changes although the API request is still sent with the same fields